### PR TITLE
Sshshell raw io

### PR DESCRIPTION
**Pull request by greg-latuszek**

### CODE REVIEW

File: moler/config/connections.py [hunk 1]
```diff
@@ -125,13 +145,33 @@ def tcp_thd_conn(port, host='localhost', name=None, **kwargs):  # kwargs to pass
                               port=port, host=host, **kwargs)  # TODO: add name
         return io_conn
 
+    def sshshell_thd_conn(host=None, port=None, username=None, login=None, password=None, name=None,
+                          reuse_ssh_of_shell=None, **kwargs):
+        mlr_conn = mlr_conn_utf8_with_clean_vt100(moler_conn_class, name=name)
+        if reuse_ssh_of_shell:
+            assert (host is None) and (port is None) and (username is None) and (login is None) and (password is None)
```
Comment (marcin-usielski) [inline moler/config/connections.py:152]: Asserts can be disabled in run time. So the conditions will not be checked.

File: moler/device/proxy_pc2.py [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+        self._use_proxy_pc = self._should_use_proxy_pc(sm_params, PROXY_PC)
+        base_or_final_state = PROXY_PC if self._use_proxy_pc else base_state
+        initial_state = initial_state if initial_state is not None else base_or_final_state
+        super(ProxyPc2, self).__init__(name=name, io_connection=io_connection,
+                                       io_type=io_type, variant=variant,
+                                       io_constructor_kwargs=io_constructor_kwargs,
+                                       sm_params=sm_params, initial_state=initial_state,
+                                       lazy_cmds_events=lazy_cmds_events)
+        self._prompt_detector_timeout = 0.5
+        self._after_open_prompt_detector = None
```
Comment (marcin-usielski) [inline moler/device/proxy_pc2.py:75]: What do you think to add to all `__init__` for devices log with information like "It's an experimental implementation of ssh device. This functionality will be merged to other device in Moler 2.0.0 and discontinued from this class" ? (or something like this)

File: moler/config/connections.py (hunk 1 above)
Comment (greg-latuszek) [reply moler/config/connections.py:152]: you are right. done

File: moler/device/proxy_pc2.py (hunk 2 above)
Comment (greg-latuszek) [reply moler/device/proxy_pc2.py:75]: good idea

Comment (greg-latuszek) [reply moler/device/proxy_pc2.py:75]: Solution shows following warnings in logs:

```
Class ProxyPc2 is an experimental/temporary implementation of device utilizing sshshell connection. It's functionality will be merged into ProxyPc device in Moler 2.0.0 and ProxyPc2 will be removed.

Class UnixRemote2 is an experimental/temporary implementation of device utilizing sshshell connection. It's functionality will be merged into UnixRemote device in Moler 2.0.0 and UnixRemote2 will be removed.

Class AdbRemote2 is an experimental/temporary implementation of device utilizing sshshell connection. It's functionality will be merged into AdbRemote device in Moler 2.0.0 and AdbRemote2 will be removed.
```

**Pull request merged**